website/src/menu-content.tsx

```diff
@@ -8,169 +8,159 @@
  * SPDX-License-Identifier: EPL-2.0
  ********************************************************************************/
 
-import * as React from 'react';
```

Comment:
Looks like this patch is adding an extra space at the beginning of each line which makes it impossible for us to review the diff for this patch.

Reply:
I've removed the leading spaces, so that you can see the changes.